src/Hazelcast.Net/Core/Clock.cs

```diff
@@ -84,7 +84,7 @@ internal static void Reset()
         /// </summary>
         /// <remarks>The epoch time in milliseconds.</remarks>
         public static long Milliseconds
-            => ToEpoch(DateTime.UtcNow);
+            => ToEpoch(Now);
```

Comment:
Just to be sure. I know we use it almost everywhere but how global is this?

Reply:
as long as the `Now` property returns `DateTime.UtcNow` it should not break anything